index.js

```diff
@@ -142,7 +233,31 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+
+  let movie = "";
+
+  if(movies.length < 1){
+    return null;
+  }
+
+  for(let i = 0; i < movies.length; i++){
+
+    let bigMula = 0;
+    let mula = movies[i].boxOffice.split('$');
+
+    // console.log(mula);
+  for(let j = 0; j < mula.length; j++){
+    if(mula[j] > bigMula){
+      bigMula = mula[j];
+    }
+  }
+  movie = movies[i].title;
+
```

Comment:
Couple of issues here - can we do more to get our `mula` value?  This number still has `,`'s in it.  Also - regardless of what we do with the `mula` variable - we set `movie` to equal `movies[i].title`.  We don't want to update unless the value is larger.  Take another shot!